test-resources/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "one-login-test-resources",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "license": "MIT",
+  "scripts": {
+    "deploy-test-resources-to-dev": "cd ../helper-scripts && ./deploy_test_resources.sh",
+    "format": "./node_modules/.bin/prettier --write '**/*.ts'",
+    "format:check": "./node_modules/.bin/prettier --check '**/*.ts'",
+    "lint": "./node_modules/.bin/eslint ./testDequeue",
+    "test:dequeue": "./node_modules/.bin/jest --testPathPattern=testDequeue/ --silent"
+  },
+  "engines": {
+    "node": "20"
+  },
+  "devDependencies": {
+    "@eslint/js": "^9.15.0",
```

Comment:
Please may we go without the carets for now - one to speak about in a ways of working session

Reply:
Removed.